fuse_models/include/fuse_models/parameters/odometry_2d_publisher_params.h

```diff
@@ -126,12 +127,14 @@ struct Odometry2DPublisherParams : public ParameterBase
     }
 
     nh.getParam("topic", topic);
+    nh.getParam("acceleration_topic", acceleration_topic);
 
     fuse_core::loadCovarianceOptionsFromROS(ros::NodeHandle(nh, "covariance_options"), covariance_options);
   }
 
   bool publish_tf { true };
   bool predict_to_current_time { false };
+  bool predict_with_acceleration { true };
```

Comment:
From my tests, when I use the acceleration it seems I get worse results in the prediction, because I have some delay.

Maybe it's better to default this to `false` :thinking:

I'm also open to changing the name of this param if you don't like it.

Reply:
The name seems fine. After reviewing the robot_localization package, I feel like it uses the acceleration data in its prediction. So setting the default to True seems correct to me.

Reply:
Actually, with `predict_with_acceleration` enabled, in my setup, likely due to delays, using acceleration in prediction gives worse results.

Not sure if that's enough a good reason to default it to `false` instead of `true`. I could always set the param to `false` in my setup, but it might be the same in most cases for other people.

Personally, I'm leaning towards the most conservative option, which is setting the default to `false`.

Thoughts?

Reply:
I don't actually care. Tom says he has no strong opinion either. So as the contributor of this new feature, I defer to you.

Reply:
Ok. I've disabled `predict_with_acceleration` by default.